docs/api/graphql/operation.mdx

```diff
@@ -55,12 +55,16 @@ const root = {
 graphql.operation(async (req, res, ctx) => {
   const payload = await graphqlRequest(
     schema,
-    req.body,
+    req.body.query,
     root,
     null,
-    req.variables,
+    req.body.variables,
   )
 
-  return res(payload)
+  if (payload.errors) {
```

Comment:
You don't necessarily have to check for errors, as in GraphQL `errors` will be undefined if there are none. Containing `errors: undefined` in the response is okay, as far as I know. 

That's to the point that we can illustrate something like this for the sake of simplicity:

```js
const { data, errors } = payload

return res(
  ctx.data(data),
  ctx.errors(errors),
)
```

Reply:
![Screenshot 2020-10-20 at 09 10 50](https://user-images.githubusercontent.com/922348/96558863-31920b00-12b4-11eb-8222-bbb194dd3731.png)
Typescript doesn't really like this on `ctx.errors`, should I send a PR fixing it?

Reply:
https://github.com/mswjs/msw/pull/430
Already did it, I think it make sense 😃